File: Selenium/Activities/Activity2.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

service = FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service) as driver:

    driver.get("https://training-support.net/selenium/login-form")

    print("Title of Login page: ", driver.title)

    driver.find_element(By.XPATH, "//input[@id='username']").send_keys("admin")

    driver.find_element(By.XPATH, "//input[@id='password']").send_keys("password")

    driver.find_element(By.XPATH, "//button[text()='Log in']").click()
    expected = driver\
        .find_element(By.XPATH, "//div[@id='action-confirmation']")\
        .text
    print(expected)

    driver.close()

File: Selenium/Activities/Activity2.java
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity2 {
    public static void main(String[] args) {
        WebDriverManager.firefoxdriver().setup();

        WebDriver driver = new FirefoxDriver();

        driver.get("https://training-support.net/selenium/login-form");
        System.out.println("Title of Login page: " + driver.getTitle());
        driver.findElement(By.xpath("//input[@id='username']"))
                .sendKeys("admin");
        driver.findElement(By.xpath("//input[@id='password']"))
                .sendKeys("password");
        driver.findElement(By.xpath("//button[text()='Log in']"))
                .click();
        String expected = driver.findElement(By
                        .xpath("//div[@id='action-confirmation']"))
                        .getText();
        System.out.println(expected);
        driver.close();




    }
}
